packages/playground/blueprints/src/lib/steps/apply-wordpress-patches/wp-content/mu-plugins/0-playground.php

```diff
@@ -21,32 +21,44 @@ function () {
  * 
  * https://github.com/WordPress/wordpress-playground/issues/498
  */
-add_filter('plugins_api_result', function ($res) {
-	if ($res instanceof WP_Error) {
+
+ // I tried to use a more semantic markup on the message below, but it messes up the display of the error message in the Plugins' Popular tags section.
+
+add_action('init', 'networking_disabled');
+function networking_disabled() {
+	global $networking_err_msg;
+	$networking_err_msg = 'Network access is an <a href="https://github.com/WordPress/wordpress-playground/issues/85">experimental, opt-in feature</a>, which means you need to enable it to allow Playground to access the Plugins/Themes directories.<br><br>
+	There are two alternative methods to enable global networking support:<br>
+	- Using the <a href="https://wordpress.github.io/wordpress-playground/query-api">Query API</a>: for example, https://playground.wordpress.net/?networking=yes; <em>or</em><br>
+	- Using the <a href="https://wordpress.github.io/wordpress-playground/blueprints-api/data-format/#features">Blueprint API</a>: add <code>"features": { "networking": true }</code> to the JSON file.
+	<br><br>
+	When browsing Playground as a standalone instance, you can enable networking via the settings panel: select the option "Network access (e.g. for browsing plugins)" and hit the "Apply changes" button.<br>
+	Please note: This option is hidden when browsing Playground as an embedded iframe.';
+}
+
+add_filter( 'plugins_api_result', 'networking_disabled_plugins' );
+ function networking_disabled_plugins($res) {
+    global $networking_err_msg;
+    if ($res instanceof WP_Error) {
 		$res = new WP_Error(
 			'plugins_api_failed',
-			'Enable networking support in Playground settings to access the Plugins directory. Network access is an <a href="https://github.com/WordPress/wordpress-playground/issues/85">experimental, opt-in feature</a>. If you don\'t want to use it, you can still upload plugins or install them using the <a href="https://wordpress.github.io/wordpress-playground/query-api">Query API</a> (e.g. ?plugin=coblocks).'
+			$networking_err_msg
```

Comment:
What do you think about calling the function here instead?

```suggestion
			networking_disabled()
```

My rationale is it's less moving parts and no risk a random plugin would override that global variable.

Reply:
That's actually why I asked about [the 'themes_api_failed'](https://developer.wordpress.org/reference/functions/themes_api/) :-)) Then we could use an array of errors, [like here](https://developer.wordpress.org/reference/classes/wp_error/).

I tested it yesterday and tried to trace the problem but couldn't get it to work. Not sure if it's a Playground blocker or my lack of knowledge...

Reply:
Gotcha! I remember I couldn't get any other method to work, although I'm hazy on the details. We could get this one in and leave the `themes_api_failed` for another PR, let's just replace a global variable with a function call :)

Reply:
Sure thing :-)